presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/api/ControlAPI.java

```diff
@@ -53,9 +53,7 @@ public ControlAPI(PrestoRpmController controller)
     @ApiOperation(value = "Start presto")
     @ApiResponses(value = {
             @ApiResponse(code = 202, message = "Acknowledged request"),
-            @ApiResponse(code = 404, message = "Presto is not installed"),
-            @ApiResponse(code = 500, message = "Failed to start presto")
```

Comment:
As all the operation will throw 500, we will document that in a more general place.